frontend/components/Main/MainRewards.tsx

```diff
@@ -65,13 +75,16 @@ const DisplayRewards = () => {
               <Text strong style={{ fontSize: 20 }}>
                 {balanceFormat(totalOlasStakedBalance, 2)} OLAS
               </Text>
-              {isStaked ? null : <Tag color="processing">Not yet staked</Tag>}
+              {minimumStakedAmountRequired && (
```

Comment:
might be a bit easier to read
```suggestion
              {minimumStakedAmountRequired && !isStaked && <Tag color="processing">Not yet staked</Tag>}
```